modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/spi/reference/AnnouncementsDeliveryTableReferenceDefinition.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.change.tracking.internal.spi.reference;
```

Comment:
I think this should be in the announcements-web module. We should also move `AnnouncementsEntryTableReferenceDefinition` and `AnnouncementsFlagTableReferenceDefinition`.